frontend/pages/workers/timerWorker.ts

```diff
@@ -1,17 +1,12 @@
-const WorkerTrans = {
-  TRANSACTIONS: "TRANSACTIONS",
-  ASSETS: "ASSETS",
-};
-
 // eslint-disable-next-line no-restricted-globals
 const timerCode = () => {
   self.onmessage = () => {
-    self.postMessage(WorkerTrans.TRANSACTIONS);
+    self.postMessage("TRANSACTIONS");
```

Comment:
Why did it change for strings?

Reply:
Because worker not recognize the when is used as variable, returns error that is not defined